matRad_calcDoseInit.m

```diff
@@ -6,6 +6,28 @@
     calcDoseDirect = false;
 end
 
+% assing analytical mode
+if isfield(pln.propDoseCalc,'fineSampling') && strcmp(pln.radiationMode, 'protons')
+    anaMode = 'fineSampling';
```

Comment:
Not completely happy about the name "anaMode" for this variable. what about pbCalcMode?

Reply:
Will change the name from anaMode to pbCalcMode.